packages/lesswrong/server/scripts/algoliaExport.js

```diff
@@ -68,3 +68,15 @@ Vulcan.runAlgoliaExport = () => {
   algoliaExport(Users, 'test_users')
   algoliaExport(Sequences, 'test_sequences')
 }
+
+Vulcan.runAlgoliaExportForCollection = (collectionName) => {
+  if (collectionName === "Posts") {
```

Comment:
Style nitpick: this would look better as a column-formatted switch statement.

Reply:
I thought about it, and didn't actually like it more